File: domain/src/main/java/no/nav/melosys/domain/dokument/soap/inntekt/inntektstype/YtelseFraOffentlige.java
package no.nav.melosys.domain.dokument.soap.inntekt.inntektstype;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlType;

import no.nav.melosys.domain.dokument.soap.inntekt.Inntekt;

@XmlType(name = "YtelseFraOffentlige")
public class YtelseFraOffentlige extends Inntekt {

    @XmlElement(required = true)
    protected String beskrivelse; // http://nav.no/kodeverk/Kodeverk/YtelseFraOffentligeBeskrivelse

    @Override
    public String getBeskrivelse() {
        return beskrivelse;
    }

    public void setBeskrivelse(String value) {
        this.beskrivelse = value;
    }
}

File: domain/src/main/java/no/nav/melosys/domain/dokument/inntekt/inntektstype/YtelseFraOffentlige.kt
package no.nav.melosys.domain.dokument.inntekt.inntektstype;

import no.nav.melosys.domain.dokument.inntekt.Inntekt;
import org.jetbrains.annotations.NotNull;

public class YtelseFraOffentlige extends Inntekt {

    protected String beskrivelse; // http://nav.no/kodeverk/Kodeverk/YtelseFraOffentligeBeskrivelse

    @Override
    @NotNull
    public String getBeskrivelse() {
        return beskrivelse;
    }

    @Override
    public void setBeskrivelse(@NotNull String beskrivelse) {
        this.beskrivelse = beskrivelse;
    }
}
